.github/workflows/c-cpp.yml

```diff
@@ -13,5 +13,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: make tests
-      run: make tests
+    - name: Run tests with gcc
+      run: make tests TOOLCHAIN=gcc
+    - name: Run tests with clang
+      run: make tests TOOLCHAIN=clang
```

Comment:
not sure if your tests always rebuild everything or not, but you may want to do `make clean tests` to remove any files build by gcc (so that make rebuilds them with clang).

Reply:
ahah the first line was:

```sh
rm -rf tests/*.o
```

but was commented! thanks!